boards/renesas/ek_ra2l1/doc/index.rst

```diff
@@ -0,0 +1,156 @@
+of serial communication interfaces, highly accurate converters and timers
+
+The key features of the EK-RA2L1 board are categorized in three groups as follow:
+
+**MCU Native Pin Access**
+
+- 48MHz Arm Cortex-M23 based RA2L1 MCU in 100 pins, LQFP package
+- Native pin access through 1 x 40-pin and 3 x 20-pin male headers
+- MCU current measurement points for precision current consumption measurement
+- Multiple clock sources - Low-precision clocks are available internal to the RA MCU
+  Additionally, RA MCU oscillator and sub-clock oscillator crystals, 20.000 MHz and 32,768 Hz,
+  are provided for precision
+- Internal MCU DC-DC power option
+
+**System Control and Ecosystem Access**
+
+- Two 5 V input sources
+
+  - USB (Debug)
+  - External power supply (using surface mount clamp test points and power input vias)
+
+- Three Debug modes
+
+  - Debug on-board (SWD)
+  - Debug in (ETM, SWD and JTAG)
+  - Debug out (SWD)
+
+- User LEDs and buttons
+
+  - Three User LEDs (red, blue, green)
+  - Power LED (white) indicating availability of regulated power
+  - Debug LED (yellow) indicating the debug connection
+  - Two User buttons
+  - One Reset button
+
+- Five most popular ecosystems expansions
+
+  - Two Seeed Grove system (I2C/Analog) connectors
+  - SparkFun Qwiic connector
+  - Two Digilent Pmod (SPI and UART) connectors
+  - Arduino (Uno R3) connector
+  - MikroElektronika mikroBUS connector
+
+- MCU boot configuration jumper
+
+.. figure:: ek-ra2l1-board.webp
+	:align: center
+	:alt: RA2L1 Evaluation Kit
+
+	EK-RA2L1 Board Functional Area Definitions (Credit: Renesas Electronics Corporation)
+
+Hardware
+********
+Detail Hardware feature for the RA2L1 MCU group can be found at `RA2L1 Group User's Manual Hardware`_
+
+.. figure:: ra2l1-block-diagram.webp
+	:width: 442px
+	:align: center
+	:alt: RA2L1 MCU group feature
+
+	RA2L1 Block diagram (Credit: Renesas Electronics Corporation)
+
+Detail Hardware feature for the EK-RA2L1 MCU can be found at `EK-RA2L1 - User's Manual`_
+
+Supported Features
+==================
+
+The below features are currently supported on Zephyr OS for EK-RA2L1 board:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock control        |
++-----------+------------+----------------------+
+
+Other hardware features are currently not supported by the port.
+
+Programming and Debugging
+*************************
+
+Applications for the ``ek_ra2l1`` board configuration can be
```

Comment:
board target

Reply:
@nordicjm @thaoluonguw 
Sorry guys, but this is already 3rd(?) PR related to the Renesas RA2, wouldn't it be better to stop the PRs wars?

Reply:
@avolkov-1221 this is the planned activity because we plan to support all RA device series within this year. Your PR is still in the discussion so this PR will be suspended. The status is draft.

Reply:
> @nordicjm @thaoluonguw Sorry guys, but this is already 3rd(?) PR related to the Renesas RA2, wouldn't it be better to stop the PRs wars?

Pardon?